pandas/core/arrays/masked.py

```diff
@@ -428,10 +428,14 @@ def to_numpy(
                     "for this dtype."
                 )
             # don't pass copy to astype -> always need a copy since we are mutating
-            data = self._data.astype(dtype)
+            with warnings.catch_warnings():
+                warnings.filterwarnings("ignore")
```

Comment:
It would be good to now what type of warnings these `warnings.filterwarnings("ignore")` are

Reply:
All runtime warnings iirc, but can add